source/10-compliance/03-counter-reporting-for-consortia.rst

```diff
@@ -7,38 +7,52 @@
 
+* Report providers MUST offer summary reports that show total usage for all members of the consortium rolled up to the consortia level, and which cannot be broken down by institution. The totals on the summary report may differ from the sum of the totals on individual member reports for the same items if an authentication method used identifies to multiple member sites and usage it attributed to each such site (e.g. overlapping IP ranges).
+* Report providers may elect to offer detailed reports that show total usage for all members of the consortium rolled up to the consortia level, and which can be broken down by institution. As with the summary report, the totals on the detailed report may differ from the sum of the totals on individual member reports for the same items.
+* Institutional reports are the individual institutional-level reports. Consortia managers and administrators MUST be able to retrieve institutional reports.
 
-Privacy and Confidentiality
-"""""""""""""""""""""""""""
+Note that consortia-level reports are restricted to COUNTER Reports. The Standard Views of the COUNTER Reports are not available as consortia-level reports.
 
-COUNTER acknowledges that some organizations treat their usage data as sensitive and private information. Content providers may include the option for consortium members to opt-out of consortium reporting. COUNTER recommends the default setting for an organization is to opt-in to consortium reporting.
 
+Creating Detailed Consortia-Level Reports
+"""""""""""""""""""""""""""""""""""""""""
 
-Content to Report Usage On
-""""""""""""""""""""""""""
+COUNTER recognizes that consolidated reports can be helpful to consortia, and that some report providers may want to meet the needs of their consortium customers by providing detailed reports. This can be done using existing extensions to the Code of Practice (see :numref:`reserved-elements`):
 
-When a COUNTER report is harvested by a consortium administrator, a content provider may choose to limit member usage to include only content acquired through the consortium. Note that when such a limitation is in place the resulting report may differ from the member-sites own version of the report. Since not all content providers can provide such limits, the consortium will be responsible for ensuring usage is filtered to the content they license for members.
+* The Customer_ID used in a detailed report MUST match that used in a report to the individual institution, and the ID for the institution returned by the /members COUNTER_SUSHI API path.
+* If provided, the Institution_Name used in a detailed report should match that used in a report to the individual institution, and the name for the institution returned by the /members COUNTER_SUSHI API path.
+* If provided, Institution_Name should be the first column in the tabular report, with Customer_ID the second column. Otherwise Customer_ID should be the first column. 
```

Comment:
These are MUST rules according to Section 11.5.

Reply:
I thought MUST wasn't being applied to if/or situations?

Reply:
Providing the reports is optional, but we expect report providers to follow the rules for reserved extensions if they use them (and the general rules for all COUNTER reports), same as we expect a report provider that optionally delivers IR to follow the rules for IR. If they don't want to follow the rules for the reserved extensions, they could create their own proprietary extensions, but that would make things more difficult for consortia that want to use the reports.